safe_transaction_service/tokens/migrations/0006_auto_20220214_1629.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.12 on 2022-02-14 16:29
+
+from django.db import migrations, models
+
+import gnosis.eth.django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tokens", "0005_add_logo_file_20220201_1335"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="token",
+            name="copy_price",
+            field=gnosis.eth.django.models.EthereumAddressV2Field(
+                help_text="If provided, copy the price from the token", null=True
+            ),
+        ),
+        migrations.AlterField(
+            model_name="token",
+            name="decimals",
+            field=models.PositiveSmallIntegerField(
+                blank=True,
+                db_index=True,
+                help_text="Number of decimals. For ERC721 tokens decimals must be `None`",
```

Comment:
Help text changes trigger db migrations, interesting

Reply:
Yes, I don't like that...